tests/originalTests/bson_test.py

```diff
@@ -4,6 +4,7 @@
 import json
 import pickle
 import unittest
+import pytest
```

Comment:
The rename is in very poor taste.  Who namesThingsInUnreadableWords.  Prefer `readable-dashes-and-lowercase-names` because lisp and unix do it that way, and it's simply in better taste.